Simplified/Book/UI/NYPLBookCellDelegate.h

```diff
@@ -19,6 +19,7 @@ shared cover registry, shared download center, et cetera. */
 @property (nonatomic) NYPLBook *book;
 #if FEATURE_AUDIOBOOKS
 @property DefaultAudiobookManager *manager;
+@property NSTimer *audiobookProgressSavingTimer;
```

Comment:
since this is not used outside of the class, can we move it to a class extension in the `.m` ?

Reply:
The timer is now part of the `AudiobookManager` object, and it will be released/cancelled when the audiobook manager is released from memory.